Tag components

FebernalGober opened the pull request:
Tag Component

# Description
The component was created to display property Tags using the React Native Paper library The component includes a function to add Tags and remove them


## Screenshots
![Captura 1](https://github.com/BrightCoders-Institute/proyecto-final-team1a/assets/126432234/455d7a5e-b7b7-4734-b4b5-ded352d947e0)
![Captura 2](https://github.com/BrightCoders-Institute/proyecto-final-team1a/assets/126432234/c1c05dd3-aafe-4da3-9d1c-a206363dc745)

FebernalGober committed:
Tag components

The component was created to display property Tags using the React Native Paper library
The component includes a function to add Tags and remove them

AlbertoG22 reviewed (approved):
![lgtm](https://github.com/BrightCoders-Institute/proyecto-final-team1a/assets/47892591/58448524-4ccc-436e-aaef-a1750e118e78)

FebernalGober committed:
Logic fix in child component

The code was modified to separate the logic that will later be used in the parent component
